Detectors/TRD/workflow/src/TRDTrackletTransformerSpec.cxx

```diff
@@ -35,6 +35,12 @@ void TRDTrackletTransformerSpec::run(o2::framework::ProcessingContext& pc)
 {
   LOG(INFO) << "Running tracklet transformer";
 
+  mTransformer.loadCalibrationParameters(mTimestamp);
```

Comment:
Just one more point: Could you please move this line to TRDTrackletTransformerSpec::init()? At the moment the timestamp anyway never changes and otherwise a new object is fetched from the CCDB for every time frame which is a quite large overhead.